PluginTemplateProject/pom.xml

```diff
@@ -131,6 +131,11 @@
                 <groupId>org.scala-tools</groupId>
                 <artifactId>maven-scala-plugin</artifactId>
                 <version>2.15.2</version>
+                <configuration>
+                    <excludes>
+                        <exclude>com/alpine/templates/db/*</exclude>
+                    </excludes>
+                </configuration>
```

Comment:
It prevents the template code from being compiled into the jar.  That way, it doesn't pollute the final jar and add an identity operator class whether they want it or not!